### ISSUE TITLE
DISC compatibility for FAVE-extract

### ISSUE THREAD

**Issue by scjs** [JoFrhwld/FAVE#11]
Update FAVE-extract for compatibility with DISC (CELEX) phonetic alphabets for use with non-North American dialects

**Comment by chrisbrickhouse** [comment]
@JoFrhwld was this resolved in f5b7e709b0bc7ac680b66626afc2bdf77bc90945? If so this can be closed.

**Comment by JoFrhwld** [comment]
No, I think these were commits for windows compatibility. The heavy lifting for allowing DISC codes is spread across a few functions, but key among them is [`isVowel()`](https://github.com/JoFrhwld/FAVE/blob/ce2c5cfa2279f771e083532ce467db167f538873/fave/extractFormants.py#L1000) which still references the global [`VOWELS`](https://github.com/JoFrhwld/FAVE/blob/ce2c5cfa2279f771e083532ce467db167f538873/fave/extractFormants.py#L99) variable.

**Comment by JoFrhwld** [comment]
Also,  a ton of plotnik.py is implicated.
